noark-extraction-validator/src/main/java/com/documaster/validator/reporting/ExcelReport.java

```diff
@@ -114,6 +118,51 @@ private void createSheets() {
 		ExcelUtils.freezePanes(summary, summaryTableRowIndex + 1, 0);
 	}
 
+	private void createExecutionInfoSheet() {
+
+		Sheet executionInfoSheet = workbook.createSheet("Execution");
+
+		// Information section row
+		Row infoSectionRow = ExcelUtils.createRow(25, executionInfoSheet);
+		ExcelUtils.createCell("Information", 0, styles.get(StyleName.GROUP), infoSectionRow);
+
+		// General Info
+		for (Map.Entry<String, Object> generalInfoEntry : getConfig().getExecutionInfo().getGeneralInfo().entrySet()) {
+			Row row = ExcelUtils.createRow(executionInfoSheet);
+			ExcelUtils.createCell(generalInfoEntry.getKey(), 0, row);
+			ExcelUtils.createCell(generalInfoEntry.getValue(), 1, row);
+		}
+
+		// Empty row
+		ExcelUtils.createRow(executionInfoSheet);
+
+		// Parameters section row
+		Row parametersSectionRow = ExcelUtils.createRow(25, executionInfoSheet);
+		ExcelUtils.createCell("Parameters", 0, styles.get(StyleName.GROUP), parametersSectionRow);
+
+		// Parameters headers
+		Row parametersHeaderRow = ExcelUtils.createRow(executionInfoSheet);
+		ExcelUtils.createCell("Name", 0, styles.get(StyleName.RESULT_HEADER_ROW), parametersHeaderRow);
+		ExcelUtils.createCell("Value", 1, styles.get(StyleName.RESULT_HEADER_ROW), parametersHeaderRow);
+		ExcelUtils.createCell("Description", 2, styles.get(StyleName.RESULT_HEADER_ROW), parametersHeaderRow);
+		ExcelUtils.createCell("Required", 3, styles.get(StyleName.RESULT_HEADER_ROW), parametersHeaderRow);
+
+		// Parameters
+		for (Command.ParameterInfo parameter : getConfig().getExecutionInfo().getParameterInfo()) {
+			if (parameter.isDefault()) {
```

Comment:
Since there are more than a dozen parameters that can be specified, if we include the default ones the execution report might get cluttered with obsolete information (especially considering the fact that most of them might even be null).